### PR TITLE
feat: insert with database

### LINKED ISSUE #1808: Support db in `insert` statement
`insert into test.t3 values('b',1122);`

### PR DESCRIPTION
**Pull request by dl239**
support insert with database 
insert into test.t3 values('b',1122);

### CODE REVIEW

File: src/sdk/sql_cluster_router.cc [hunk 1]
```diff
@@ -363,27 +363,23 @@ bool SQLClusterRouter::GetMultiRowInsertInfo(const std::string& db, const std::s
                                              std::vector<uint32_t>* str_lengths) {
     if (status == NULL || table_info == NULL || default_maps == NULL || str_lengths == NULL) {
         status->msg = "insert info is null";
-        LOG(WARNING) << status->msg;
```
Comment (vagetablechicken) [inline src/sdk/sql_cluster_router.cc]: If no warning log, it's hard to debug. We have some confused msgs, e.g. `"can't find column " + col_name + " in table "`, `col_name` is a distraction. May be we can make a func or macro, e.g. 
```
RETURN_WITH_WARN(code, msg):
    status.msg = msg
    log(warn) << msg
    return status
```

Comment (dl239) [reply src/sdk/sql_cluster_router.cc]: we can do this in #656.

File: src/sdk/sql_cluster_router.cc [hunk 2]
```diff
@@ -2321,11 +2321,6 @@ std::shared_ptr<hybridse::sdk::ResultSet> SQLClusterRouter::ExecuteSQL(const std
                 return {};
             }
 
-            // TODO(denglong): Should support table name with database name
-            if (db.empty()) {
-                *status = {::hybridse::common::StatusCode::kCmdError, "Please use database first"};
-                return {};
-            }
             if (!ExecuteInsert(db, sql, status)) {
```
Comment (vagetablechicken) [inline src/sdk/sql_cluster_router.cc:2324]: Plz make a comment for `ExecuteInsert`, arg1 `db` can be empty, just a default db. If sql contains db info, this one won't be used.

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**